pkg/kritis/review/validating_transport_test.go

```diff
@@ -37,6 +37,7 @@ func encodeB64(in string) string {
 
 func TestValidatingTransport(t *testing.T) {
 	successSec, pub := testutil.CreateSecret(t, "test-success")
+	_, pub2 := testutil.CreateSecret(t, "test-success-2")
```

Comment:
Could you please add a high-level comment on why 2 secrets are created? And just a brief description of the purpose of the test and its setup.

Reply:
This is for testing any key logic in validating attestor. The attestor will have two good keys, and the attestation's signature is verified by one of them.
Also adding some comments in test for clarification.